exercises/sort_by_pattern_test.rb

```diff
@@ -46,23 +47,44 @@ def test_sort_by_distance
   end
 
   def test_sort_by_length
-    skip
     words = ["heteromorph", "ancyloceratina", "bioengineering", "mathematical", "bug"]
-    # Your code goes here
+    transformed = []
+    words.each do |word|
+      transformed << [word.length, word]
+    end
+    transformed.sort!
+    sorted = []
+    transformed.each do |sort_key, word|
```

Comment:
Unused block argument - `sort_key`. If it's necessary, use `_` or `_sort_key` as an argument name to indicate that it won't be used.